etcd/v3/kv_etcd.go

```diff
@@ -1484,9 +1483,15 @@ func (et *etcdKV) RemoveMember(
 	}
 	et.kvClient.SetEndpoints(newClientUrls...)
 	et.maintenanceClient.SetEndpoints(newClientUrls...)
+
+	return et.removeMember(removeMemberID)
```

Comment:
could removeMemberID be 0 here? Should we return an error in that case?

nit: we could use this function just to figure out the removeMemberID then call RemoveMemberByID() to avoid some code duplication.

Reply:
removeMemberID would be 0 only if we could not find a member for the given name and IP. The `removeMember` function handles `ErrMemberNotFound`. But I will add an extra check here to return success if removeMemberID == 0